src/test/kotlin/graphics/scenery/tests/examples/advanced/EyeTrackingExample.kt

```diff
@@ -1,10 +1,11 @@
 package graphics.scenery.tests.examples.advanced
 
+import cleargl.GLVector
 import org.joml.Vector3f
 import graphics.scenery.*
 import graphics.scenery.backends.Renderer
 import graphics.scenery.controls.OpenVRHMD
-import graphics.scenery.controls.eyetracking.PupilEyeTracker
+import graphics.scenery.controls.eyetracking.PupilEyeTrackerNew
```

Comment:
Main changes: the New file no longer contains screen space calibration, only world space. A distance calculation was added to the Gaze object. Reference points were decreased to 12 instead of 18, and samples per point decreased to 70 (from 120).

Reply:
Okay then - please replace PupilEyeTracker with PupilEyeTrackerNew 👍